README.md

```diff
@@ -33,8 +33,8 @@ const fastify = require('fastify')()
 
 fastify.register(require('@fastify/cookie'), {
   secret: "my-secret", // for cookies signature
-  addHookOnRequest: false, // disable cookie parsing on the onRequest phase
-  parseOptions: {}     // options for parsing cookies
+  hook: false, // disable cookie autoparsing or set autoparsing on any of the hooks: 'onRequest', 'preHandler', 'preParsing', 'preValidation', 'preSerialization'. default: onRequest
```

Comment:
```suggestion
  hook: false, // disable cookie autoparsing or set autoparsing on any of the hooks: 'onRequest', 'preHandler', 'preParsing', 'preValidation', 'preSerialization'. default: 'onRequest'
```

Reply:
I think to exclude 'preSerialization' and 'PreRapsing', they have different arguments in the handler

Reply:
please set it `'onRequest'` here.